chain-cli/chaincode-template/e2e/dex-api.spec.ts

```diff
@@ -302,6 +368,54 @@ describe("DEx v3 Testing", () => {
       expect(data?.liquidity).toEqual("2060.753664493334613554");
     });
 
+    test("Adding liquidity more than max liquidity will throw error", async () => {
+      const pa = 1700,
+        pb = 1900;
+      const [ta, tb] = spacedTicksFromPrice(pa, pb, tickSpacing);
+      const slippage = 0.5;
+      const token0 = new BigNumber("0"),
+        token1 = new BigNumber("10000000000000000000000000000000000000000000000000");
+      const [token0Slipped, token1Slipped] = slippedValue([token0, token1], slippage);
+
+      const dto = new AddLiquidityDTO(
+        ETH_ClassKey,
+        USDT_ClassKey,
+        fee,
+        ta,
+        tb,
+        token0,
+        token1,
+        token0Slipped,
+        token1Slipped
+      ).signed(user.privateKey);
+
+      const addLiquidityRes = await client.dexV3Contract.addLiquidity(dto);
+      expect(addLiquidityRes.Message).toBe("liquidity crossed max liquidity");
+    });
+    test("Adding liquidity equal to zero will throw error", async () => {
+      const pa = 1700,
+        pb = 1900;
+      const [ta, tb] = spacedTicksFromPrice(pa, pb, tickSpacing);
+      const slippage = 0.5;
+      const token0 = new BigNumber("0"),
+        token1 = new BigNumber("0");
+      const [token0Slipped, token1Slipped] = slippedValue([token0, token1], slippage);
+
+      const dto = new AddLiquidityDTO(
+        ETH_ClassKey,
+        USDT_ClassKey,
+        fee,
+        ta,
+        tb,
+        token0,
+        token1,
+        token0Slipped,
+        token1Slipped
+      ).signed(user.privateKey);
+
+      const addLiquidityRes = await client.dexV3Contract.addLiquidity(dto);
+      expect(addLiquidityRes.Message).toBe("Invalid Liquidity");
+    });
```

Comment:
```suggestion
    });

```

Improved readability